src/cpl.rs

```diff
@@ -95,15 +191,22 @@ impl CslStringList {
         self.len() == 0
     }
 
-    /// Get an iterator over the name/value elements of the list.
-    pub fn iter(&self) -> CslStringListIterator {
+    /// Get an iterator over the `name=value` elements of the list.
+    pub fn iter(&self) -> impl Iterator<Item = CslStringListEntry> + '_ {
```

Comment:
It might be better to return `CslStringListIterator` here, just in case the user wants to name that type.

Although, with generators coming "soon", this expectation might change (because they're can't be named anyway).